app/templates/views/announcements.cy.html

```diff
@@ -100,7 +100,7 @@ <h1 class="govuk-heading-xl">
     {% endif %}
     {% if loop.first %}
       <h2 class="govuk-heading-m {% if not date_loop.first %}govuk-!-margin-top-9{% endif %}">
-        {{ alert_or_planned_test.starts_at_date.date_as_lang }}
+        {{ alert_or_planned_test.starts_at_date.datetime_as_lang }}
```

Comment:
I think macro loop around this breaks down the planned tests by date.  As such there's no guarantee the second planned test will be at the same time as the first planned test

Reply:
The page is only designed to show one planned alert, with the sections allowing the announcement to be different for the different countries of the UK.  If the page needs to be modified to allow multiple planned alerts on different dates to be shown, this will need to go through UI prototyping first.